common/membership/hashring.go

```diff
@@ -275,6 +282,41 @@ func (r *ring) ring() *hashring.HashRing {
 	return r.value.Load().(*hashring.HashRing)
 }
 
+func (r *ring) emitHashIdentifier() float64 {
+	members, err := r.peerProvider.GetMembers(r.service)
```

Comment:
does `GetMembers` return full list including self? If not we should add it

Reply:
I depled and tested, yes

Reply:
I don't know to be honest, but also I think a view of the hashring that excludes the current machine is a valid view (if it's drained, for example), and you still want to ensure it's consistent across both drained and undrained zones. 

I would not want to mutate the membership list to add this because it would make a drain case look like a split when it wasn't in this case

Reply:
If each host sees a list of members that doesn't include themselves then the hashed value will never match. Am I missing something?

Reply:
Maybe I'm missing something? 

If I'm on host `10.0.0.1` and I have a view of the hashrhing as equivalent to the hashed version of 
```
10.0.0.2
10.0.0.3
```
then, this should be the same as the view for `10.0.0.2` and 3 respectively? The fact that it's not actually on the active list is not a problem?

Reply:
the only reason I fetched self, was to add it as a dimension for the guage to prevent interleaving of the metric